src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -35,6 +35,17 @@ class DropdownAutoCompleteMenu extends React.Component {
       ),
     ]),
 
+    /**
+     * If this is undefined, autocomplete filter will use this value instead of the
+     * current value in the filter input element.
+     *
+     * This is useful if you need to strip characters out of the search
+     */
+    filterValue: PropTypes.string,
```

Comment:
I updated the comments a bit (the "undefined" logic was backwards). It's not really a `default`-InputValue because it should change as the input value changes (a parent component should be controlling the value based on changes of the filter input).

Reply:
alrighty sgtm

Reply:
ah I see—so this is more of a way to operate on the input string. That makes sense. I think something like `inputOverride` would have made more sense to me but maybe 🤷‍♀️ since it already went out.